GitUtils.ps1

```diff
@@ -115,12 +115,15 @@ function Get-GitStatus($gitDir = (Get-GitDirectory)) {
         $filesModified = @()
         $filesDeleted = @()
         $filesUnmerged = @()
+        $stashCount = 0
 
         if($settings.EnableFileStatus -and !$(InDisabledRepository)) {
             dbg 'Getting status' $sw
             $status = git -c color.status=false status --short --branch 2>$null
+            $stashCount = $null | git stash list 2>$null | measure-object | select -expand Count

```

Comment:
Well I'm stumped... I can confirm the swallowed input issue, and the fix, but I can't reproduce for any other Git command (`reflog` seeming like it may be most similar). Very odd.